app/imports/collections/applicationSections.js

```diff
@@ -0,0 +1,13 @@
+import { Meteor } from 'meteor/meteor';
+
+class ApplicationSectionsCollection extends Mongo.Collection {
+  insert(doc, callback) {
+    doc.createdAt = doc.createdAt || new Date();
+    doc.updatedAt = doc.updatedAt || new Date();
+    doc.userId = Meteor.userId();
+    doc.year = doc.year || new Date().getFullYear();
```

Comment:
i was thinking this could be a data to diff check next year. maybe we could set an `env` var to set the year/edition of the same fair/event/whateber. Im not completly sure about this to approach but it;s not a bad idea to leave it there for now ,no?